pandas/tests/indexes/test_base.py

```diff
@@ -909,6 +907,13 @@ def test_symmetric_difference(self):
         expected = MultiIndex.from_tuples([('bar', 2), ('baz', 3), ('bar', 3)])
         assert tm.equalContents(result, expected)
 
+        # on equal multiIndexs
```

Comment:
Can you move this to it's own dedicated test, and add a comment with a link to the github issue?